### PR TITLE
Small fixes related to rollback

### PR DESCRIPTION
**Pull request by marimano**
Make some properties of commands public to have a possibility to check command execution in Reach

### CODE REVIEW

File: src/DynamoCore/Models/DynamoModelCommands.cs [hunk 1]
```diff
@@ -202,6 +202,7 @@ void MakeConnectionImpl(MakeConnectionCommand command)
         void BeginConnection(Guid nodeId, int portIndex, PortType portType)
         {
             bool isInPort = portType == PortType.Input;
+            activeStartPort = null;
```
Comment (marimano) [inline src/DynamoCore/Models/DynamoModelCommands.cs]: if `EndConnection` failed, `activeStartPort` will not be reset

**Comment by pboyer** [comment]
Hi @marimano!

If it's not too much trouble, can you add a regression test to DynamoCoreTests for the fix for EndConnection?  

Other than that, this looks fine.

**Comment by marimano** [comment]
Hi @pboyer,
I added that test case, but it fails because view layer is present. I'd like to upload it after `MAGN-7277 Where possible, change tests in DynamoCoreTests to use DynamoModelTestBase` will be done and therefore to leave this PR as it is. 
Don't you mind?

**Comment by pboyer** [comment]
Hi @marimano,

That's fine.  Can you please create a task to track creating this regression test?

**Comment by pboyer** [comment]
@marimano I have to revert this due to test regressions:

1) Test Failure : Dynamo.Tests.FileLibraryTests.Directory_Delete
     Expected: False
  But was:  True

at Dynamo.Tests.FileLibraryTests.Directory_Delete() in c:\EC\Dynamo\Dynamo\test\Libraries\CoreNodesTests\FileTests.cs:line 207

**Comment by marimano** [comment]
@pboyer, I just run the tests, and everything is okay with this test case:
dir_delete

**Pull request merged**